codes/models/SRRaGAN_model.py

```diff
@@ -374,8 +396,12 @@ def optimize_parameters(self, gen, step):
                     p.requires_grad = True
 
                 l_d_total = 0
-                pred_d_real = self.netD(self.var_ref)
-                pred_d_fake = self.netD(self.fake_H.detach())  # detach to avoid BP to G
+                if self.use_frequency_separation:
+                        pred_d_real = self.netD(self.filter_high(self.var_ref))
+                        pred_d_fake = self.netD(self.filter_high(self.fake_H.detach())) # detach to avoid BP to G
```

Comment:
Reduce indentation to match the rest of the code

Reply:
Oops...